### PR TITLE
Add featured talks and meetups section

### PR DESCRIPTION
**Pull request by staceypotter**
Added invite for Leigh's GOTK guide walk-through on Oct 19 (10am PT/18:00 BST) to the Community Section. LMK if this should be somewhere else - or condensed (wasn't sure how much info to include). Thanks! :)

### CODE REVIEW

**Comment by stefanprodan** [comment]
@staceypotter I think this should under a different category, something like "Featured Talks" where we could list the KubeCon talk and others with a link to Youtube then the one you posted.

**Comment by staceypotter** [comment]
Sounds great, thanks Stefan. I'll make that change. :)

**Comment by staceypotter** [comment]
@stefanprodan - I added the "featured talks" section as well as a header link. :) Thanks!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # GitOps Toolkit
 
+(Check out our [Upcoming Meetups](#upcoming-meetups)!)
```
Comment (stefanprodan) [inline README.md:3]: Please remove this line

**Review by stefanprodan — CHANGES_REQUESTED** [review]

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @staceypotter 🥇

File: README.md (hunk 1 above)
Comment (staceypotter) [reply README.md]: I removed it. But is there a way to put *something* on the top half of the page that directs people downward to the talks/meetups? We want people to be able to see these things before getting lost in a wall of text...afterall they're informative, educational videos, not just marketing "fluff."

(Check out our [Upcoming Meetups](#upcoming-meetups)!)

**Comment by dholbach** [comment]
We could also add this to fluxcd/community?

**Pull request merged**